python/tvm/relay/backend/vm.py

```diff
@@ -212,7 +212,6 @@ def __init__(self, mod, ctx, target):
         self.vm.init(ctx)
 
     def _make_executor(self, expr=None):
-        assert expr is None
```

Comment:
`expr` is never used in below...

Reply:
`expr` is the interface of `_make_executor`, so we have to keep it though not using it, because caller will pass in `expr`.